decidim-participatory_processes/app/types/decidim/participatory_processes/participatory_process_type.rb

```diff
@@ -6,13 +6,45 @@ module ParticipatoryProcesses
     ParticipatoryProcessType = GraphQL::ObjectType.define do
       interfaces [
         -> { Decidim::Core::ParticipatorySpaceInterface },
+        -> { Decidim::Core::ScopableInterface },
         -> { Decidim::Core::AttachableInterface }
       ]
 
       name "ParticipatoryProcess"
       description "A participatory process"
 
+      field :slug, !types.String
```

Comment:
Could you please add specs for all these fields below? You can find all the current specs for a `ParticipatoryProcess` here: https://github.com/decidim/decidim/blob/master/decidim-participatory_processes/spec/types/participatory_process_type_spec.rb

Reply:
Thanks for reviewing, I wasn't sure how to add those specs. I'll make the changes and get back to you.